detections/endpoint/windows_unusual_mozilla_nss_mozglue_access.yml

```diff
@@ -0,0 +1,73 @@
+name: Windows Unusual Mozilla NSS-Mozglue Access
+id: 1a7e7650-b81d-492e-99d4-d5ab633afbdd
+version: 1
+date: '2025-07-16'
+author: Teoderick Contreras, Splunk
+status: production
+type: Anomaly
+description: The following analytic identifies processes loading Mozilla NSS-Mozglue libraries
+  such as mozglue.dll and nss3.dll. It leverages Sysmon Event logs, specifically monitoring
+  EventCode 7, which tracks image loaded events. This activity is significant because
+  it can indicate unauthorized access or manipulation of these libraries, which are
+  commonly used by Mozilla applications like Firefox and Thunderbird. If confirmed malicious,
+  this could lead to data exfiltration, credential theft, or further compromise of the system.
+data_source:
+- Sysmon EventID 7
+search: '`sysmon` EventCode=7  ImageLoaded IN ("*\\mozglue.dll", "*\\nss3.dll") 
+  NOT(process_name IN("firefox.exe", "thunderbird.exe")) NOT(process_path IN("*\\program files*")) 
```

Comment:
nss3 can also be used by other processes. For example Code42 `C:\Program Files\Code42\nlib\nss3.dll`

For `mozglue`. I think its also worth filtering additional Mozilla products https://www.mozilla.org/en-US/products/

maybe install the most known and add filters